imagefiles/dockcross

```diff
@@ -180,7 +180,7 @@ fi
 # Now, finally, run the command in a container
 #
 tty -s && TTY_ARGS=-ti || TTY_ARGS=
-CONTAINER_NAME=dockcross
+CONTAINER_NAME=dockcross_$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 7 | head -n 1)
```

Comment:
The `$(cat ...` doesn't work on the Mac.

`$RANDOM` is available in bash, so the script will work across platforms:

  https://coderwall.com/p/s2ttyg/random-number-generator-in-bash